ams/api/src/test/java/com/netease/arctic/api/metrics/MetricDefineTest.java

```diff
@@ -43,7 +44,7 @@ public void testEquals(MetricDefine target, boolean expectedEquality) {
       assertEquals(source.hashCode(), target.hashCode(), "MetricNames hash code should be equal");
     } else {
       assertNotEquals(source, target, "MetricNames should not be equal");
-      assertNotEquals(
+      Assertions.assertNotEquals(
```

Comment:
Is this change necessary?

Reply:
This should be auto-generated when optimizing imports
I will fix it.